modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -14,6 +14,8 @@ import java.util.Date;
 
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
 
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
```

Comment:
@MicaelleSilvaa, let's talk about it offline because I checked the `replacements.json` and it's not following the order from this file neither the alphabetical order, therefore I'm confuse. :smile:

Reply:
Resolved :)